integration/spark-common/src/main/scala/org/apache/spark/sql/execution/command/carbonTableSchemaCommon.scala

```diff
@@ -47,6 +47,7 @@ import org.apache.carbondata.spark.util.DataTypeConverterUtil
 
 case class TableModel(
     ifNotExistsSet: Boolean,
+    var databaseName: String,
```

Comment:
There is already a databaseNameOp in line 51

Reply:
fixed